ntbs-service/Services/MockMigrationRepository.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ntbs_service.DataMigration;
+using ntbs_service.DataMigration.RawModels;
+
+namespace ntbs_service.Services
```

Comment:
This is added here because the other mock service are. Which actually brings up a point I overlooked: in the config we have the option to turn off legacy search, in that case I'm guessing we should also use this service in the app?

Reply:
As you've seen, the other mock services that are in the `ntbs-service` project are there because they can be configured for use in the real app. For those services, my best guess as the reason it's set up this way is that in the early days of the project they may have been developing features that required the specimen matching database before it had been developed, or at least before it was available in all environments. We should probably remove this config at some point because I can't see it ever being used again.

The legacy search config appears to be different - it looks as though this is intended to be a truly configurable feature... it's possible that there is or was thought to be a business reason to turn off the legacy search. I think this because the wording is different (i.e. "enabled" vs "mock out") and because that config option only disables legacy search - it doesn't disable all migration database features (e.g. bulk migration).

This class, on the other hand, was introduced to help with integration tests, so belongs in the integration test project. We don't currently have a reason to make it configurable for real app use (and given the number of exceptions it throws, it would seem strange if we ever did) so we shouldn't.